charts/plex-media-server/values.yaml

```
@@ -22,9 +22,13 @@ ingress:
   # -- The url to use for the ingress reverse proxy to point at this pms instance
   url: ""
 
-  # -- Optional secret name to provide valid https connections
+  # -- Optional TLS configuration to provide valid https connections
   # using an existing SSL certificate
-  certificateSecret: ""
+  # tls:
+  #   - hosts:
+  #       - plex.example.com
+  #     secretName: ""
+  tls: []
```

nit: place example config values below the var definition.

```suggestion
  tls: []
  #   - hosts:
  #       - plex.example.com
  #     secretName: example-dot-com
```

---

Hi @cilindrox thanks for the feedback I've pushed another commit with your suggestions.

-Brian